src/main/scala/org/broadinstitute/dsde/workbench/sam/api/AdminRoutes.scala

```diff
@@ -284,4 +296,33 @@ trait AdminRoutes extends SecurityDirectives with SamRequestContextDirectives wi
       user.id,
       samRequestContext
     )
+
+  private def getSamUserSupportSummary(workbenchEmail: WorkbenchEmail, samRequestContext: SamRequestContext) =
+    for {
+      samUserOption <- userService.getUserFromEmail(workbenchEmail, samRequestContext)
+      samUser = samUserOption.getOrElse(throw new RuntimeException("user not found"))
+      allowances <- userService.getUserAllowances(samUser, samRequestContext)
+      maybeAttributes <- userService.getUserAttributes(samUser.id, samRequestContext)
+      termsOfServiceDetails <- tosService.getTermsOfServiceDetailsForUser(samUser.id, samRequestContext)
+      enterpriseFeatures <- resourceService
+        .listResourcesFlat(
+          samUser.id,
+          Set(ResourceTypeName("enterprise-feature")),
+          Set.empty,
+          Set(ResourceRoleName("user")),
+          Set.empty,
+          includePublic = false,
+          samRequestContext
+        )
+      directGroupMemberships <- userService.countDirectGroupMemberships(samUser, samRequestContext)
+      indirectGroupMemberships <- userService.countIndirectGroupMemberships(samUser, samRequestContext)
+    } yield SamUserSupportSummaryResponse(
+      samUser,
+      allowances,
+      maybeAttributes,
+      termsOfServiceDetails.getOrElse(TermsOfServiceDetails(None, None, permitsSystemUsage = false, isCurrentVersion = false)),
+      Map("enterpriseFeatures" -> enterpriseFeatures.toJson),
+      directGroupMembershipCount = directGroupMemberships,
+      indirectGroupMembershipCount = indirectGroupMemberships
+    )
```

Comment:
This follows the pattern set by `getSamUserCombinedState` in [UserRoutesV2](src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV2.scala), in which business logic that requires multiple services is implemented in the route class.

Reply:
I would put this in `userService` (which already knows about `tosService`).

Reply:
Unfortunately `UserService` doesn't know about `ResourceService`, which we need to get the enterprise features. Whaddya think - is it worth giving `UserService` that knowledge, or do we not actually care about the enterprise features at this point?

Reply:
Moved to `UserService`. This is kinda hacky in that the method requires the caller to pass in a `ResourceService`. I am hesitant to change the `UserService` constructor just for this one API.